app/Http/Controllers/ArtworkController.php

```diff
@@ -80,7 +81,9 @@ public function forsalePost(Request $request)
         'dimension' => $validatedData['dimension'],
         'price' => $validatedData['price'],
         'users_id' => Auth::id(),
-        'category_id' => $validatedData['category_id']
+        'category_id' => $validatedData['category_id'],
+        'status' => 'pending', // Default status
```

Comment:
Same here. Can you check if you can remove this. I believe you already set a default status value on database migration config.